backend/segmentation/tests/unit/test_api_segmentation.py

```
@@ -10,272 +25,279 @@
     async def test_segment_image_large_file(self, async_client: AsyncClient):
-        """Test segmentation with file that exceeds size limit."""
-        # Create a large dummy image (simulate large file)
-        large_image = Image.new('RGB', (5000, 5000), color='white')
-        image_buffer = io.BytesIO()
-        large_image.save(image_buffer, format='JPEG', quality=95)
-        large_image_bytes = image_buffer.getvalue()
-        
-        files = {"image": ("large.jpg", io.BytesIO(large_image_bytes), "image/jpeg")}
-        data = {"model_name": "hrnet"}
-        
+        """Test segmentation with a large image file (5000×5000).
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)